go.mod

```diff
@@ -1,26 +1,24 @@
-module github.com/cyriltovena/logger
+module cluster-logging-load-client/loadclient
```

Comment:
Modules need always to be URL, e.g. `module github.com/ViaQ/loki-operator`. In this case `module github.com/ViaQ/cluster-logging-load-client`

Reply:
Fixed
DONE